### PR TITLE
chore: add initial simple loadgen

### PR DESCRIPTION
**Pull request by cprice404**
This commit adds an initial port of the simple load generator
from the python/js SDKs.  It will need some additional error
handling, and we need to add a client-side max TPS flag, but
should be usable for initial tuning work.

### CODE REVIEW

File: Momento.Sdk.sln [hunk 1]
```diff
@@ -5,9 +5,11 @@ VisualStudioVersion = 16.0.810.11
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Momento.Sdk", "src\Momento.Sdk\Momento.Sdk.csproj", "{188A76AD-422A-4829-B59F-9EC1C102EA38}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Momento.Sdk.Tests.Unit", "tests\Unit\Momento.Sdk.Tests\Momento.Sdk.Tests.csproj", "{77C34767-7796-4842-BE75-20563DB3230A}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Momento.Sdk.Tests.Unit", "tests\Unit\Momento.Sdk.Tests\Momento.Sdk.Tests.Unit.csproj", "{77C34767-7796-4842-BE75-20563DB3230A}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Momento.Sdk.Tests.Integration", "tests\Integration\Momento.Sdk.Tests\Momento.Sdk.Tests.csproj", "{FB56314E-BDCF-474A-A2CD-8956934FF11B}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Momento.Sdk.Tests.Integration", "tests\Integration\Momento.Sdk.Tests\Momento.Sdk.Tests.Integration.csproj", "{FB56314E-BDCF-474A-A2CD-8956934FF11B}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Momento.Sdk.Incubating.Tests", "tests\Integration\Momento.Sdk.Incubating.Tests\Momento.Sdk.Incubating.Tests.csproj", "{A0F6221C-42A9-454B-BB82-EDB9841DF6E1}"
```
Comment (malandis) [inline Momento.Sdk.sln:12]: Didn't we remove the incubating tests?

File: examples/MomentoLoadGen/Program.cs [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+
+        static async Task Main(string[] args)
+        {
+            using ILoggerFactory loggerFactory = InitializeLogging();
+
+            CsharpLoadGeneratorOptions loadGeneratorOptions = new CsharpLoadGeneratorOptions(
+              /**
+               * Configures the Momento client to timeout if a request exceeds this limit.
+               * Momento client default is 5 seconds.
+               */
```
Comment (malandis) [inline examples/MomentoLoadGen/Program.cs:354]: Try using triple-front-slash `///` for the C# docstrings

File: examples/MomentoLoadGen/Program.cs [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+        }
+
+
+        private async Task LaunchAndRunWorkers(
+            SimpleCacheClient client,
+            CsharpLoadGeneratorContext context,
+            int workerId,
+            int numOperations)
+        {
+            for (var i = 1; i <= numOperations; i++)
```
Comment (eaddingtonwhite) [inline examples/MomentoLoadGen/Program.cs:135]: do we want to add the concept of max concurrency like we had in other basic load generators? Not sure how un controlled concurrency effects latency in c# vs js or python but assuming still some latency impact on having no limit on this?

Comment (eaddingtonwhite) [reply examples/MomentoLoadGen/Program.cs:135]: Sorry missed `numberOfConcurrentRequests` below looks good 👍

**Review by eaddingtonwhite — APPROVED** [review]
LGMT for first iteration 🚢 . Why no f# version :trollface:

File: examples/MomentoLoadGen/Program.cs (hunk 2 above)
Comment (cprice404) [reply examples/MomentoLoadGen/Program.cs:354]: thanks gtk

File: examples/MomentoLoadGen/MomentoLoadGen.csproj [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  <ItemGroup>
+    <None Remove="Microsoft.Extensions.Logging" />
+    <None Remove="Microsoft.Extensions.Logging.Console" />
+    <None Remove="Momento.Sdk" />
+    <None Remove="HdrHistogram" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
+    <PackageReference Include="Momento.Sdk" Version="0.27.1" />
```
[the remark this answers is not in the document]
Comment (cprice404) [reply examples/MomentoLoadGen/MomentoLoadGen.csproj:25]: cool yeah i will update next time i'm in there but I don't think there are any perf-impacting changes in recent releases.

**Pull request merged**